vespalib/src/vespa/vespalib/util/resource_limits.h

```diff
@@ -0,0 +1,23 @@
+// Copyright Yahoo. Licensed under the terms of the Apache 2.0 license. See LICENSE in the project root.
+
+#pragma once
+
+#include <cstdint>
+
+namespace vespalib {
+
+/*
+ * Class for getting resource limits.
```

Comment:
Please add some details on how the resource limits are sampled.

Reply:
Done.